docs/guides/use_checkpointing.md

```diff
@@ -193,46 +219,52 @@ serialization.from_state_dict(custom_target, raw_state_dict)
+# Refer to https://jax.readthedocs.io/en/latest/multi_process.html#initializing-the-cluster
+# for how to set it up in multi-process scenarios.
+jax.distributed.initialize("localhost:8889", num_processes=1, process_id=0)
 
-```{code-cell} ipython3
-am = checkpoints.AsyncManager()
+async_checkpointer = orbax.AsyncCheckpointer(orbax.PyTreeCheckpointHandler(), timeout_secs=50)
 
 # Mimic a training loop here:
 for step in range(2, 3):
-    checkpoints.save_checkpoint(ckpt_dir, ckpt, step=2, overwrite=True, keep=3, async_manager=am)
+    checkpoints.save_checkpoint(ckpt_dir, ckpt, step=2, overwrite=True, keep=3,
+                                orbax_checkpointer=async_checkpointer)
     # ... Continue with your work...
 
 # ... Until a time when you want to wait until the save completes:
-am.wait_previous_save()  # Block until the checkpoint saving is completed.
+async_checkpointer.wait_until_finished()  # Blocks until the checkpoint saving is completed.
 checkpoints.restore_checkpoint(ckpt_dir, target=None, step=2)
 ```
 
 ## Multi-host/multi-process checkpointing
 
 JAX provides a few ways to scale up your code on multiple hosts at the same time. This usually happens when the number of devices (CPU/GPU/TPU) is so large that different devices are managed by different hosts (CPU). To get started on JAX in multi-process settings, check out [Using JAX in multi-host and multi-process environments](https://jax.readthedocs.io/en/latest/multi_process.html).
 
-In the [Single Program Multi Data (SPMD)](https://jax.readthedocs.io/en/latest/glossary.html#term-SPMD) paradigm with JAX [`pjit`](https://jax.readthedocs.io/en/latest/jax.experimental.pjit.html), a large multi-process array can have its data sharded across different devices (check out the `pjit` [JAX-101 tutorial](https://jax.readthedocs.io/en/latest/jax-101/08-pjit.html)). This kind of data array needs a special experimental JAX API—[`GlobalAsyncCheckpointManager`](https://github.com/google/jax/blob/c7dcd0913bc8b4878a7e45184553c331254b801a/jax/experimental/gda_serialization/serialization.py#L452)—to save and restore checkpoints. This API lets each host dump its data shards to a single shared storage, such as a Google Cloud bucket.
+In the [Single Program Multi Data (SPMD)](https://jax.readthedocs.io/en/latest/glossary.html#term-SPMD) paradigm with JAX [`pjit`](https://jax.readthedocs.io/en/latest/jax.experimental.pjit.html), a large multi-process array can have its data sharded across different devices (check out the `pjit` [JAX-101 tutorial](https://jax.readthedocs.io/en/latest/jax-101/08-pjit.html)). This kind of data array needs a special experimental JAX API — [`gda_serialization`](https://github.com/google/jax/blob/c7dcd0913bc8b4878a7e45184553c331254b801a/jax/experimental/gda_serialization/serialization.py) — to save and restore checkpoints. This API lets each host dump its data shards to a single shared storage, such as a Google Cloud bucket.
```

Comment:
Maybe we should rename `gda_serialization` -> `array_serialization` (Reference: @yashk2810 's PR `jax.Array` github.com/google/flax/pull/2703)